stdlib/LinearAlgebra/test/generic.jl

```diff
@@ -225,6 +225,11 @@ end
     end
 end
 
+@testset "Issue #30466" begin
+    @test norm([typemin(Int), typemin(Int)], Inf) == 9.223372036854776e18
```

Comment:
`Int` is `Int32` on 32 bit systems so make the rhs here `float(typemax(Int))` and `2float(typemax(Int))` in the line below.

Reply:
You mean `-float(typemin(Int))`, I guess (since `typemin` and `typemax` are not equal in magnitude).